docs/index.rst

```diff
@@ -17,38 +17,34 @@ Supported Network Operating Systems:
 * Cisco IOS
 * Cisco IOS-XR
 * Cisco NX-OS
-* Fortinet Fortios
-* IBM
 * Juniper JunOS
-* Mikrotik RouterOS
-* Palo Alto NOS
-* Pluribus
-* Vyos 
+
+extras
+______
+
+In addition to the core drivers napalm also supports community driven drivers. You can find more information about them here:
+
+TBD Link to extras/core drivers' documentation
+
+Selecting the right driver
+--------------------------
 
 You can select the driver you need by doing the following:
 
 .. code-block:: python
 
-   >>> from napalm_base import get_network_driver
+   >>> from napalm import get_network_driver
    >>> get_network_driver('eos')
-   <class napalm_eos.eos.EOSDriver at 0x10ebad6d0>
+   <class napalm.eos.eos.EOSDriver at 0x10ebad6d0>
```

Comment:
`napalm.eos.EOSDriver` instead of `napalm.eos.eos.EOSDriver`; this seems to be everywhere like that

Reply:
```
(reunification) ➜  napalm git:(reunification_docs) python
Python 2.7.14 (default, Sep 25 2017, 09:53:22)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from napalm import get_network_driver
>>> get_network_driver('eos')
<class 'napalm.eos.eos.EOSDriver'>
```